ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.scss

```diff
@@ -14,15 +14,8 @@
   flex-direction: column;
   gap: 10px;
 
-  .yasgui {
-
-    .autocompleteWrapper {
-      display: none !important;
-    }
-
-    .tabContextButton {
-      display: none;
-    }
+  .hidden-control-bar {
```

Comment:
Can we just use the `hidden` class we already have instead of adding another one with the exact same purpose.